### PR TITLE
Added Admin panel entry for all models

### LINKED ISSUE #258: Models are not connected with admin.py
Only the User model seems to be connected with admin.py, while we have volunteer, organization, job etc still not connected to admin.py. 

https://docs.djangoproject.com/en/1.9/ref/contrib/admin/

### PR DESCRIPTION
**Pull request by amruthasangeeth**
Fix #258

### CODE REVIEW

File: vms/vms/templates/vms/base.html [hunk 1]
```diff
@@ -49,16 +49,40 @@
...
                         {% if user.volunteer %} 
                             <li><a href="{% url 'shift:view_volunteer_shifts' user.volunteer.id %}">{% trans "Upcoming Shifts" %}</a></li>
                             <li><a href="{% url 'shift:view_hours' user.volunteer.id %}">{% trans "Completed Shifts" %}</a></li>
                             <li><a href="{% url 'event:list_sign_up' user.volunteer.id %}">{% trans "Shift Sign Up" %}</a></li>
                             <li><a href="{% url 'volunteer:report' user.volunteer.id %}">{% trans "Report" %}</a></li>
-                            <li><a href="{% url 'volunteer:profile' user.volunteer.id %}">{% trans "Profile" %}</a></li>
+
+            <ul class="nav navbar-nav navbar-right">
+                        <li class="dropdown">
+                            <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false"
```
Comment (tapasweni-pathak) [inline vms/vms/templates/vms/base.html]: @amruthasangeeth Please remove these changes from this PR. Please send another PR for these changes. Raise an issue if you don't see a related one. I'll be happy to merge this PR without these changes as these are not related to the issue we are referring to, in this PR.

Comment (amruthasangeeth) [inline vms/vms/templates/vms/base.html]: I have removed the changes occurred in base.html.

**Comment by tapasweni-pathak** [comment]
@amruthasangeeth If I merge this PR, two unnecessary commits will go to the commit log. If someone tries to go through the commit log to understand the changes, this will add up to extra work.

Please remove the two unwanted commits. Only have the relevant diff in this pull request. Rebase and push to update this PR. 

Let me know if you get stuck.

**Comment by tapasweni-pathak** [comment]
@amruthasangeeth Can you please clean this up?

**Comment by amruthasangeeth** [comment]
@tapasweni-pathak I have cleaned the unwanted commits

**Comment by tapasweni-pathak** [comment]
Thanks for the work. :)

**Pull request merged**